.changelog/1997.bugfix.md

```diff
@@ -0,0 +1 @@
+Show available amount next to delegation input
```

Comment:
1996 and it's rather trivial not a bugfix

Reply:
Changed

tho it is a somewhat visible improvement ("Bug Fixes and Improvements")